### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by FallenCB** [KadeArchive/Kade-Engine#2192] [General]
How do I fix this error when compiling the game? PS C:\Users\jaide\Documents\Games\Friday Night Funkin'\Kade Engine\Source Code\Kade-Engine-stable> lime test windows
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
source/LuaClass.hx:35: characters 46-58 : Type not found : StatePointer
source/Replay.hx:55: lines 55-145 : Defined in this class
source/LuaClass.hx:1669: characters 42-54 : Type not found : StatePointer
source/Replay.hx:55: lines 55-145 : Defined in this class
source/LuaClass.hx:1469: characters 42-54 : Type not found : StatePointer
source/Replay.hx:55: lines 55-145 : Defined in this class
source/LuaClass.hx:1144: characters 42-54 : Type not found : StatePointer
source/Replay.hx:55: lines 55-145 : Defined in this class
source/LuaClass.hx:855: characters 41-53 : Type not found : StatePointer
source/Replay.hx:55: lines 55-145 : Defined in this class
source/LuaClass.hx:635: characters 40-52 : Type not found : StatePointer
source/Replay.hx:55: lines 55-145 : Defined in this class
source/LuaClass.hx:370: characters 42-54 : Type not found : StatePointer

**Comment by NK-523** [comment]
run these in command prompt
```
haxelib remove linc_luajit
haxelib remove hxvm-luajit
haxelib git linc_luajit https://github.com/nebulazorua/linc_luajit.git
haxelib git hxvm-luajit https://github.com/nebulazorua/hxvm-luajit
```

**Reply by FallenCB** [reply]
Thanks!
